README.md

```diff
@@ -0,0 +1,80 @@
+![Wagtail](https://releases.wagtail.io/wagtail-github-header.png?x)
+
+Wagtail is an open source content management system built on Django, with a strong community and commercial support. It's focused on user experience, and offers precise control for designers and developers. 
+
+![Wagtail screenshot](https://releases.wagtail.io/wagtail-screenshot-with-browser.png)
+
+### Features
+
+* A fast, attractive interface for authors
+* Complete control over design and front-end
+* Scales to millions of pages and thousands of editors
+* Fast out of the box, cache-friendly when you need it
+* Content API for 'headless' sites with de-coupled front-end
+* Runs on a Raspberry Pi or a multi-datacenter cloud platform 
+* StreamField encourages flexible content without compromising structure
+* Powerful, integrated search, using Elasticsearch or PostgreSQL
+* Excellent support for images and embedded content
+* Multi-site and multi-language ready
+* Embraces and extends Django
+
+Find out more at https://wagtail.io.
```

Comment:
The link would be better as [wagtail.io](https://wagtail.io), like the docs link further down.